src/sql/util.py

```diff
@@ -441,7 +424,7 @@ def get_user_configs(primary_path, alternate_path):
         elif section_found and data:
             return data, file_path
 
-    return data, None
+    return None, None
```

Comment:
This single line change I believe to be an error in the initial implementation which is not strictly related to #975, but was contributing to #969 and potentially could cause other issues -- the other pyproject.toml data would be returned if by this function if no condition caused an earlier return. 

I tried not to address #969 directly in this PR, but was not sure what this ought to return now that `data` is being read more cleanly. Perhaps this was meant only for the case of `~/.jupysql/config` to allow it not to require headers? That use case isn't specifically documented or tested for but it could make sense. Thoughts?

Reply:
I think you are right, this was an error in the initial implementation (made in [this PR.](https://github.com/ploomber/jupysql/pull/935)). 

> Perhaps this was meant only for the case of ~/.jupysql/config to allow it not to require headers? 

I don't believe that case was considered. Based on a conversation in that PR, the assumption was that `data` would only evaluate to either the `[tool.jupysql.SqlMagic]` config settings, `{}`, or `None`. 

I would say your change is great for now. If we want `~/.jupysql/config` to not require headers later, we can go back and change it in another PR.